packages/doenetml-worker-rust/lib-doenetml-core/src/core/state/types/content_refs.rs

```diff
@@ -72,6 +123,8 @@ const _: () = {
     // process and turn all `ContentRef` into `FlatDastElementContent` before serializing.
     #[declare]
     type ContentRef = FlatDastElementContent;
+    #[declare]
```

Comment:
Just curious why this has to be done for ContentRef and AnnotatedContentRefs but not ContentRefs.

Reply:
`ContentRefs` serializes as an array of `ContentRef`, which is the correct type for Typescript :-)